src/ldp/http/SparqlUpdateBodyParser.ts

```diff
@@ -17,8 +18,7 @@ export class SparqlUpdateBodyParser extends BodyParser {
   protected readonly logger = getLoggerFor(this);
 
   public async canHandle({ request }: BodyParserArgs): Promise<void> {
```

Comment:
Although it's hidden here since it wasn't there when this class was written, BodyParserArgs also contains a metadata object. So you should be able to extract the content-type from that instead of having to call the parser on the header.

Reply:
OK, pushed a new version that relies on metadata instead!